roles/zuul/files/jobs/lore.yaml

```diff
@@ -6,6 +6,7 @@
       - zuul-git-prep
       - shell: |
           #!/bin/bash -ex
+          sudo apt-get install -yqq nodejs npm
```

Comment:
Do we know what version of nodejs we get this way? Maybe it makes more sense to use nvm and pin?